### ISSUE TITLE
default to yellow highlighter

### ISSUE THREAD

**Issue by jdlongman** [burtonator/polar-bookshelf#743]
Present behavior: when text is selected, option to choose highlighter color is revealed.

Desired behavior: when text is selected, it is highlighted in yellow by default (most used highlight), but clicking on an **existing** highlighted text pops up the color menu, to allow for changing to a (less frequently used) red or orange highlight. This way regular highlighting is fast but multicolors can still be supported.

**Comment by burtonator** [comment]
yes... we're going to add a 'default highlighter' option and you can select popup, as well as just automatically create the highlight if you want.

Should take a few weeks though.
